### PR TITLE
Hiding advanced properties

### PR DESCRIPTION
**Pull request by MarcelS4**
Hiding/Showing of Events actions and Advanced Properties has been implemented by using the "Windowbuilder Basic" preference

Closes issue #145 

image

### CODE REVIEW

File: org.eclipse.wb.core/src/org/eclipse/wb/internal/core/editor/structure/property/ComponentsPropertiesPage.java [hunk 1]
```diff
@@ -80,25 +80,25 @@
...
   ////////////////////////////////////////////////////////////////////////////
+  @Override
   public void dispose() {
     Control control = getControl();
     if (control != null && !control.isDisposed()) {
       control.dispose();
     }
   }
 
+  @Override
```
Comment (vogella) [inline org.eclipse.wb.core/src/org/eclipse/wb/internal/core/editor/structure/property/ComponentsPropertiesPage.java:91]: I dislike mixing different concerns in one PR. Adding @override does not belong to the topic of this PR

Can you send another PR for adding the @overrides? You find clean-up action in the package explorer -> Right-mouse click -> Cleanup Missing Code tab. Please run this only with one cleanup action at the time to make review easier.

File: org.eclipse.wb.core/src/org/eclipse/wb/internal/core/editor/structure/property/ComponentsPropertiesPage.java [hunk 2]
```diff
@@ -527,16 +525,14 @@ private void update_showEventsAction() throws Exception {
...
-    ExecutionUtils.runLog(new RunnableEx() {
-      public void run() throws Exception {
-        if (m_showEvents) {
-          showEvents();
-        } else {
-          showProperties();
-        }
-        // update actions
-        updateActions();
+    ExecutionUtils.runLog(() -> {
```
Comment (vogella) [inline org.eclipse.wb.core/src/org/eclipse/wb/internal/core/editor/structure/property/ComponentsPropertiesPage.java:528]: Please send other separate PR for the annymous class to lambda conversion. You can do this via another cleanup, tab Code Style.

**Comment by MarcelS4** [comment]
Please see PR #157 instead
